### PR TITLE
Improve docker install docs

### PR DESCRIPTION
**Pull request by Nutomic**

### CODE REVIEW

File: assets/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      - PICTRS__MEDIA__ANIMATION__MAX_HEIGHT=256
+      - PICTRS__MEDIA__ANIMATION__MAX_FRAME_COUNT=400
+    user: 991:991
+    volumes:
+      - ./volumes/pictrs:/mnt:Z
+    restart: always
+    logging: *default-logging
+
+  postgres:
+    image: postgres:16-alpine
```
Comment (dessalines) [inline assets/docker-compose.yml:70]: Use `pgautoupgrade/pgautoupgrade:16-alpine`

File: assets/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      - ./nginx_internal.conf:/etc/nginx/nginx.conf:ro,Z
+      - ./proxy_params:/etc/nginx/proxy_params:ro,Z
+    restart: always
+    logging: *default-logging
+    depends_on:
+      - pictrs
+      - lemmy-ui
+
+  lemmy:
+    image: dessalines/lemmy:0.19.5
```
Comment (dessalines) [inline assets/docker-compose.yml:24]: We're gonna need to remember to up these manually.

**Review by dessalines — APPROVED** [review]
Looks good. The ansible one has gotten too template-y now anyway.

File: assets/docker-compose.yml (hunk 2 above)
Comment (Nutomic) [reply assets/docker-compose.yml:24]: I think we had some docs for the release process somewhere, but mosty ignore it. Maybe echo a message about upgrading these in the backend release script?

**Review by dessalines — APPROVED** [review]

**Pull request merged**